src/components/governance/PageProposal.vue

```diff
@@ -167,6 +191,22 @@ export default {
     proposal() {
       return this.proposals.proposals[this.proposalId]
     },
+    getNextProposalId() {
+      let id = this.getProposalIndex(`+1`)
+      if (id !== undefined) {
+        return id
+      } else {
+        return this.proposalId
```

Comment:
you could just return `id` and check in the view `v-show = "getPrevProposalId"`
 `

Reply:
would reduce lines of code by a bunch